pom.xml

```diff
@@ -103,6 +103,18 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I am reading from https://quarkus.io/guides/micrometer that Micrometer is the recommended approach to metrics for Quarkus. Could you please switch the implementation to use micrometer instead? You should probably only add these 2 dependencies:
```
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-micrometer</artifactId>
</dependency>
<dependency>
    <groupId>io.micrometer</groupId>
    <artifactId>micrometer-registry-prometheus</artifactId>
</dependency>
```